code/src/components/AlbumCard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Icons } from './Icons';
+import ArtistName from './ArtistName';
+
+const AlbumCard = (props) => {
```

Comment:
A reflection; what is the best practice and why? When shall we pass on (props) to a component and write props.name and when shall we pass on all the names instead const AlbumCard = ({img, URL, name})?

Reply:
Best practice is to deconstruct the props like you did there in the last example, since it makes the code much easier to follow. Will update code now 🙏🏻